From: Luigi Baldoni <aloisio@gmx.com>
Date: 2018-04-24 12:45:56 +0200
Subject: Disable version check

Set variable so that there is no automated version
update check inside the program.

Index: rapid-photo-downloader-0.9.34/raphodo/constants.py
===================================================================
--- rapid-photo-downloader-0.9.34.orig/raphodo/constants.py
+++ rapid-photo-downloader-0.9.34/raphodo/constants.py
@@ -31,7 +31,7 @@ remote_versions_file = "https://damonlyn
 
 # If set to True, the ability to check for a new version will be removed
 # from the user interface and disabled in program logic.
-disable_version_check = False
+disable_version_check = True
 
 
 class CheckNewVersionDialogResult(IntEnum):
